### DISCUSSION TITLE
Format for the list of gene identifiers

### DISCUSSION THREAD

**Discussion by wamclaughlin** [zqfang/GSEApy#137] [Q&A]
Hi: Would there possibly be a way to run the gseapy.enrichr command using UniProt accession codes as input rather gene identifiers?

**Comment by zqfang** [comment]
I'm afraid no. Enrichr server only accept gene names

**Reply by wamclaughlin** [reply]
Thank you for letting me know. 
As a work around, I see that UniProt has a mapping service for going from UniProt accessions to gene names, as available at the URLs https://www.uniprot.org/uploadlists/ and https://www.uniprot.org/help/api_idmapping.

**Reply by zqfang** [reply]
you could also using bioservices's UniProt API to do this. It's in python
